konrad/cloud.py

```diff
@@ -344,6 +345,7 @@ def __init__(self, numlevels, cloud_fraction, lw_optical_thickness,
                 (for the shortwave component of RRTMG)
             single_scattering_albedo (float / DataArray): single scattering
                 albedo due to cloud (for the shortwave component of RRTMG)
+            norm_index (int / None): model level index for coupling the cloud
```

Comment:
What happens if the ``norm_index`` is actually ``None`` during runtime?

Reply:
It is set in the first iteration, equal to `norm_new`, which for the HighCloud would be the convective top index. I just wanted to add the option to set it beforehand, in case suddenly adding a cloud causes the convection to act weirdly for the first few iterations.

Reply:
Ahh I see, we don't need it in the first time step, as it is only needed for the shift, but not the initialisation. Got it 😉